.circleci/config.yml

```diff
@@ -74,7 +74,7 @@ jobs:
                 composer create-project jsdrupal/drupal-admin-ui-demo -s dev --prefer-dist
                 cd drupal-admin-ui-demo
                 mv ~/project drupal-admin-ui
-                composer config repositories.repo-name path "./drupal-admin-ui/admin_ui_support"
+                composer config repositories.repo-name path "drupal-admin-ui/admin_ui_support"
```

Comment:
@webflo Do you know how we can correctly link in a folder of the current project?

Reply:
Have you tried to not run ``mv ~/project drupal-admin-ui`` and use ``composer config repositories.repo-name path "~/project/admin_ui_support" instead?

Reply:
``../project/admin_ui_support`` should work as well according to the docs on https://getcomposer.org/doc/05-repositories.md#path